packages/mgt-chat/src/statefulClient/StatefulGraphChatClient.ts

```diff
@@ -337,34 +377,44 @@ class StatefulGraphChatClient implements StatefulClient<GraphChatClient> {
    * @memberof StatefulGraphChatClient
    */
   private async updateFollowedChat() {
-    // reset state to initial
-    this.notifyStateChange((draft: GraphChatClient) => {
-      draft.status = 'initial';
-      draft.messages = [];
-      draft.chat = undefined;
-      draft.participants = [];
-    });
-    // Subscribe to notifications for messages
-    this.notifyStateChange((draft: GraphChatClient) => {
-      draft.status = 'creating server connections';
-    });
     // avoid subscribing to a resource with an empty chatId
     if (this._chatId) {
```

Comment:
Generally speaking we should use the property and not the backing field, 
```suggestion
    if (this.chatId) {
```